games-engines/openmw/openmw-0.46.0.ebuild

```diff
@@ -0,0 +1,161 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic xdg-utils readme.gentoo-r1
+
+DESCRIPTION="Open source reimplementation of TES III: Morrowind"
+HOMEPAGE="https://openmw.org/"
```

Comment:
Please also add the Github/Gitlab page as a 2ndary homepage here, whichever upstream prefers.